clicache/src/Log.cpp

```diff
@@ -70,12 +70,6 @@ namespace Apache
           static_cast<apache::geode::client::LogLevel>(level));
       }
 
-      bool Log::Enabled(LogLevel level)
```

Comment:
Doesn't this change the public API of the .NET library?

Reply:
Maybe?  I'll have a look.  Easy enough to fix, if so.

Reply:
CLI Log class is actually `private`, but I think it makes sense to have an `Enabled()` method anyway, so it's back now.